src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/service/SyncServiceTest.kt

```diff
@@ -384,22 +383,4 @@ class SyncServiceTest {
     // verify telemetry not sent
     verify(telemetryClient, never()).trackEvent(anyOrNull(), anyOrNull(), anyOrNull())
   }
-
-  @Test
-  fun `cannot update report during initial migration`() {
```

Comment:
this is covered by `NomisSyncResourceTest.SamplePayloads.cannot update a report during initial migration`.

this test cannot be performed here because the service is instantiated directly without spring’s proxying (and hence no input validation).